mantle/kola/harness.go

```diff
@@ -940,7 +1083,7 @@ func runTest(h *harness.H, t *register.Test, pltfrm string, flight platform.Flig
 	if err != nil {
 		h.Fatalf("Cluster failed: %v", err)
 	}
-	defer func() {
+	destroyCluster := func() {
```

Comment:
so basically - now that we run multiple tests on a single VM we can't `defer func()` here, but we need to return it to the caller and call it at the very end?

Reply:
Yep that's the idea. This change allows us to use the logic in `runTest` for non-exclusive tests.